molecule/templates/test.j2

```diff
@@ -0,0 +1,2 @@
+this is a {{ test }}
+
```

Comment:
Not really a fan of this.  I know we need to test it, but test files shouldn't live along side real files.